docs/src/Flex.doc.js

```diff
@@ -13,7 +14,7 @@ card(
   <PageHeader
     name="Flex"
     description={`
-      Flex is a layout component with a very limited subset of the props available to Box and a few special props of its own.
+      Flex is a layout component with a very limited subset of the props available to [Box](/Box) and a few special props of its own.
```

Comment:
@dangerismycat the only few special props is gap, right? should we just be direct with that?

And then say somethinglike 

Use this component for flex layouts, especially when even spacing between elements is desired by using the 'gap' property.

Reply:
Yeah, that's pretty much it. And that copy sounds good to me 👍

Reply:
Done :)